engine/src/main/java/org/terasology/world/chunks/CoreChunk.java

```diff
@@ -56,12 +56,30 @@
      */
     Block getBlock(int x, int y, int z);
 
+    /**
+     * Returns block's id at given position relative to the chunk.
+     *
+     * @param pos Position of the block relative to corner of the chunk
+     * @return Block at given position
+     */
+    short getBlockId(Vector3ic pos);
```

Comment:
Should we directly add a default implementation unwrapping the vector here?

```suggestion
    default short getBlockId(Vector3ic pos) {
        return getBlockId(pos.x(), pos.y(), pos.z());
    }
```